robolectric/src/main/java/org/robolectric/Robolectric.java

```diff
@@ -15,19 +15,18 @@
 
 public class Robolectric {
   private static final ShadowsAdapter shadowsAdapter = instantiateShadowsAdapter();
-  private static ServiceLoader<ShadowProvider> providers;
+  private static Iterable<ShadowProvider> providers;
```

Comment:
I couldn't figure out another way to test it because ServiceLoader is final and can't be mocked by Mockito. If you've got another suggestion I'm open to it.